variables.tf

```diff
@@ -2,8 +2,60 @@
 # Input Variables
 ########################################################################################################################
 
-#variable "my_variable" {
-#  type        = string
-#  description = "A description of my variable"
-#  default     = "default_value"
-#}
+variable "watsonx_data_name" {
+  type        = string
+  description = "The name of the watsonx.data instance."
+  default     = null
```

Comment:
add variable validation to ensure this is not null is user is not passing existing instance